README.md

```diff
@@ -1,2 +1,60 @@
-# -Covid-Stats
-A world wide covid stars
+# Covid stats
+
+> #### A web application for a company whcih provides commercial and scientific space travel services. In this application users can book rockets and join the space missions useing live data from the spaceX API.
+
+## Built With
+
+- React.js
+- Redux (@redux/toolkit)
+- React-Bootstrap
+
+## Additional Used
+
+- Api
+- KanbanBoard
+- Linters
+- Gitflow
+- npm
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+`git clone git@github.com:miftah1991/space-travelers.git `
+
+Then `cd space-travelers`
+
+Then `npm install` to install packages
+
+Finally `npm start`
+
+## Authors
+
+👤 **Mohamed Eldeeb**
+
+- GitHub: [@eng-mohamed-eldeeb](https://github.com/eng-mohamed-eldeeb)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohamed-eldeeb-a69022206/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).